redisinsight/ui/src/pages/autodiscover-cloud/redis-cloud-subscriptions/RedisCloudSubscriptions/RedisCloudSubscriptions.tsx

```diff
@@ -226,66 +209,55 @@ const RedisCloudSubscriptions = ({
 
   const Account = () => (
     <>
-      <span className={styles.account_item}>
-        <span className={styles.account_item_title}>Account ID:&nbsp;</span>
-        <span color="subdued" data-testid="account-id">
-          {account?.accountId ?? <LoadingContent lines={1} />}
-        </span>
-      </span>
-      <span className={styles.account_item}>
-        <span className={styles.account_item_title}>Name:&nbsp;</span>
-        <span color="subdued" data-testid="account-name">
-          {account?.accountName ?? <LoadingContent lines={1} />}
-        </span>
-      </span>
-      <span className={styles.account_item}>
-        <span className={styles.account_item_title}>Owner Name:&nbsp;</span>
-        <span color="subdued" data-testid="account-owner-name">
-          {account?.ownerName ?? <LoadingContent lines={1} />}
-        </span>
-      </span>
-      <span className={styles.account_item}>
-        <span className={styles.account_item_title}>Owner Email:&nbsp;</span>
-        <span color="subdued" data-testid="account-owner-email">
-          {account?.ownerEmail ?? <LoadingContent lines={1} />}
-        </span>
-      </span>
+      <AccountItem>
+        <AccountItemTitle>Account ID:&nbsp;</AccountItemTitle>
```

Comment:
nit: I see it was like that even before your changes, but since we already have a styled component for these titles, we can use padding properly, instead of `&nbsp;`